packages/deployments/contracts/contracts/core/connext/helpers/OZERC20.sol

```diff
@@ -229,8 +226,13 @@ contract ERC20 is IERC20, IERC20Permit, EIP712 {
 
     _beforeTokenTransfer(_sender, _recipient, amount);
 
-    balances[_sender] = balances[_sender].sub(amount, "ERC20: transfer amount exceeds balance");
-    balances[_recipient] = balances[_recipient].add(amount);
+    uint256 fromBalance = balances[_sender];
+    require(fromBalance >= amount, "ERC20: transfer amount exceeds balance");
+    unchecked {
+      balances[_sender] = fromBalance - amount;
+    }
+    balances[_recipient] += amount;
```

Comment:
Could be updated to:
```sol
        uint256 fromBalance = _balances[from];
        require(fromBalance >= amount, "ERC20: transfer amount exceeds balance");
        unchecked {
            _balances[from] = fromBalance - amount;
            // Overflow not possible: the sum of all balances is capped by totalSupply, and the sum is preserved by
            // decrementing then incrementing.
            _balances[to] += amount;
        }
```

also needs to call `_afterTokenTransfer`.

[source](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/e0137d4cbe14ba6be2b52c6ecfb006b515c6f780/contracts/token/ERC20/ERC20.sol#L226-L248)